contracts/Cruize.sol

```diff
@@ -214,8 +234,22 @@ contract Cruize is CruizeVault, Proxy {
         vaultState.queuedWithdrawShares = uint128(newQueuedWithdrawShares);
         //slither-disable-next-line reentrancy-no-eth
         currentQueuedWithdrawalShares[token] = 0;
-
         ShareMath.assertUint104(lockedBalance);
         vaultState.lockedAmount = uint104(lockedBalance);
     }
+
+    function tokensTvl(
+        address[] memory assets
+    ) external returns (uint256[] memory) {
+        uint256 length = assets.length;
+        uint256[] memory assetsTvl = new uint256[](length);
+        for (uint256 i = 0; i < length; i++) {
+            address token = assets[i];
+            assetsTvl[i] =
+                vaults[token].lockedAmount +
+                vaults[token].totalPending;
+        }
+
+        return assetsTvl;
```

Comment:
hey can you paste an example of what tokensTvl will return ?

Reply:
> hey can you paste an example of what tokensTvl will return ?

it will be a array of tvl's of given token addresses like
`[1000*10^18,2000*10^6 , 100 * 10^8]`

Reply:
how will I know which asset's TVL is on index 0 and so on ?

Reply:
if you pass USDC on index 0 while passing the tokens array in the function then the TVL for USDC will be on the 0 indexes in the  function return 
array

Reply:
the order in which we will send the input , we get the output values in the same order

Reply:
ohh okay